### PR TITLE
Update test1.py

### PR DESCRIPTION
**Pull request by sappho192**

### CODE REVIEW

File: test1.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import tensorflow as tf
-tf.set_random_seed(777)  # for reproducibility
+tf.set_random_seed(1774)  # for reproducibility
```
Comment (taekwang94) [inline test1.py:2]: Why did you change this value to 1774?
I want to maintain 777.

File: test1.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 # X and Y data
 x_train = [1, 2, 3]
+z_train = [1, 2, 3]
```
Comment (taekwang94) [inline test1.py]: Do you really need this line?

File: test1.py [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 y_train = [1, 2, 3]
 
 X = tf.placeholder(dtype=tf.float32)
 Y = tf.placeholder(dtype=tf.float32)
+Z = 3
```
Comment (taekwang94) [inline test1.py]: What is this variable ?

**Review by taekwang94 — CHANGES_REQUESTED** [review]

File: test1.py (hunk 1 above)
Comment (sappho192) [reply test1.py:2]: 1774 is better.

File: test1.py (hunk 2 above)
Comment (sappho192) [reply test1.py]: I don't think so.

File: test1.py (hunk 3 above)
Comment (sappho192) [reply test1.py]: Forget it.

Comment (taekwang94) [reply test1.py]: ok.

**Review by taekwang94 — APPROVED** [review]
